### PR TITLE
Added a security group column to the characters table

### PR DESCRIPTION
**Pull request by natrist**
This will enable the core character-wide security group features and lead the way for the GM client functionalities.

---
This change is [Reviewable](https://reviewable.io/reviews/mangostwo/database/158)

### CODE REVIEW

File: Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- Adding a security group column to the characters table:
+-- This column is used to store the character-wide security group (default is 0 for players)
+ALTER TABLE `character2`.`characters`
```
Comment (H0zen) [inline Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql]: Non portable. 'character2' is a database name. Remove it.

**Comment by Necrovoice** [comment]
Also missing mangos database version update format. When the server code portion is resolved (https://github.com/mangostwo/server/pull/162) I will update this PR to mangos standard before pushing.

File: Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql (hunk 1 above)
Comment (natrist) [reply Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql]: Thanks, but how would I go about fixing this?

**Comment by natrist** [comment]
Thank you, @Necrovoice :)

File: Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql (hunk 1 above)
Comment (billy1arm) [reply Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql]: change the line to:
ALTER TABLE `characters` 

the database for characters will normally be pre-selected by this point based on the fact it's in the character folder.

Comment (natrist) [reply Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql]: Thanks, I just did.

**Pull request closed without merging**